packages/x-codemod/README.md

```diff
@@ -56,26 +56,73 @@ through jscodeshift's `printOptions` command line argument
 npx @mui/x-codemod <transform> <path> --jscodeshift="--printOptions='{\"quote\":\"double\"}'"
 ```
 
-## Included scripts
```

Comment:
Removing this h2 allows us to have a better nested structure.

Before:

- Incldued scripts
  -  `v6.0.0`
    - `preset-safe`
  -  Pickers codemods
     - ... list all the individual codemods
  - Data grid codemods
    - ... list all the individual codemods



After:

- v7.0.0
  - `preset-safe`
  -  Pickers codemods
     - ... list all the individual codemods
  - Data grid codemods
    - ... list all the individual codemods

- v6.0.0
  - `preset-safe`
  -  Pickers codemods
     - ... list all the individual codemods
  - Data grid codemods
    - ... list all the individual codemods